### ISSUE TITLE
[Public catalog] Move quickstart logo and CTAs, make header sticky

### ISSUE THREAD

**Issue by LizBaker** [newrelic/developer-website#1800]
We want the CTA buttons for quickstarts to remain visible to users even after they scroll. 

Mobile approach: keep the same approach we already have in place.

### AC
- [x] (logo + name + main CTA + breadcrumb) are in a sticky header and as the user scrolls down, they can access the main actions at any given time.
- [x] Logo moves to the left
- [x] CTA buttons move to the right 
- [x] As the screen size is reduced, the overall size of the sticky header section reduces
- [ ] Measure before and after for data analysis so we understand the impact of this change + / -
- [x] ~~the mobile view should also have this treatment~~  -`won't do`

### Design
![SKX929djFq0XEe7b0dVskeQPsHqVb0YEe4lwOHLZJAGOzTa63g-fIPBQrCoANf4vBYLyL4kDOs-Si2sSlsTeYLaoKA3UvP5pumOTojAGrakilADT3kNvFVl3KBGZaw61oHN-vyFZ=s0](https://user-images.githubusercontent.com/39655074/138000123-f33a9045-ffe0-4d73-8024-322e746b7cf2.png)

**Comment by jpvajda** [comment]
We are waiting on some final input from @legnaleama to finish this up
